scripts/init-database.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+
+import yaml
+import sqlalchemy
+import MySQLdb
+
+values = list()
+
+
+def retreiveValues():
+    stream = open("../.config/bassa/config.yml", "r")
```

Comment:
I think we should find the path of the file and then access the parent dir for this programmatically. 

We can use following for that, 
```
import os
path = os.path.abspath(__file__)
```
See https://stackoverflow.com/a/595315/1190184

Think what will happen if someone ran this script like `python scripts/init-database.py`? What will be the `../` folder be? Mu guess is, it will be the parent dir of the Bassa project dir. 

Anyway, the config file's path should be `bassa.yml` not `.config/bassa/config.yml`